api/tests/python/nro_services/test_nro_request_utils.py

```diff
@@ -269,28 +269,27 @@ def test_add_nwpta(app, request, session, pns):
     ], False),
     ([
-        {'choice_number': 1, 'name': 'name corp1', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
-        {'choice_number': 2, 'name': 'name corp2', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
+        {'choice_number': 1, 'name': 'name corp1', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+        {'choice_number': 2, 'name': 'name corp2', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
     ], False),
     ([
-        {'choice_number': 1, 'name': 'name corp1', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
-        {'choice_number': 2, 'name': 'name corp2', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
-        {'choice_number': 3, 'name': 'name corp3', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
+        {'choice_number': 1, 'name': 'name corp1', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+        {'choice_number': 2, 'name': 'name corp2', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
     ], False),
     ([
-         {'choice_number': 1, 'name': 'name corp', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
+         {'choice_number': 1, 'name': 'name corp', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
      ], True),
     ([
-         {'choice_number': 1, 'name': 'name corp1', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
-         {'choice_number': 2, 'name': 'name corp2', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
+         {'choice_number': 1, 'name': 'name corp1', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+         {'choice_number': 2, 'name': 'name corp2', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
      ], True),
     ([
-         {'choice_number': 1, 'name': 'name corp1', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
-         {'choice_number': 2, 'name': 'name corp2', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
-         {'choice_number': 3, 'name': 'name corp3', 'designation': 'ltd', 'name_state_type_cd': 'NE'},
+         {'choice_number': 1, 'name': 'name corp1', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+         {'choice_number': 2, 'name': 'name corp2', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
+         {'choice_number': 3, 'name': 'name corp3', 'designation': 'ltd', 'name_state_type_cd': 'NE', 'consumption_date': '01-SEP-08 11:11:11', 'corp_num': 'S1234567'},
      ], True),
 ]
 
```

Comment:
All of these names are NE (not examined) so it does not make sense for them to have consumption date and corp num. 

I'd suggest leaving the existing test data as is, and adding several new test names to this list of test data (names_test_data) for approved names with consumption date and corp num.